src/toolbars/MeterToolBar.cpp

```diff
@@ -150,7 +151,7 @@ void MeterToolBar::Populate()
       //(maybe we should do it differently for Arabic language :-)  )
       mRecordSetupButton = safenew AButton(this);
       mRecordSetupButton->SetLabel({});
-      mRecordSetupButton->SetName(_("Record"));
+      mRecordSetupButton->SetName(wxString::Format(_("Record meter peak %d db"), -DecibelScaleCutoff.GetDefault()));
```

Comment:
Why GetDefault() and not just Read()?

Now if I change interface preferences so the bottom of scale is not -60 dB, the drawing updates for it correctly, but the name spoken for the button still says -60, inconsistently with the display.

Reply:
`DecibelScaleCutoff.GetDefault()` also occurs in the constructor of MeterPanel.  I put it there at commit 01a53e7da9b616d26d1bb49f381180f04438504f.  As a behavior-preserving code transformation, it was correct as I did it, eliminating the use of the constant ENV_DB_RANGE.  But was old behavior correct?  Perhaps not, and that other line should also be corrected to use Read().

Reply:
Thanks, fixed